gatsby-node.js

```diff
@@ -9,6 +9,8 @@ const axios = require('axios')
 
 const BASE_URL = 'http://localhost:5000/fetch?url='
 const HomeTemplate = path.resolve('./src/templates/HomePage.tsx')
+const AuthorTemplate = path.resolve('./src/templates/Author.tsx')
```

Comment:
@peterbaile look into syntactical sugar to make these imports nicer? Or is the best we can do